bugreportutils/bugreportutils.go

```diff
@@ -137,6 +153,11 @@ type SensorInfo struct {
 	Version, Number int32
 	TotalTimeMs     int64 // time.Duration in Golang is converted to nanoseconds in JS, so using int64 and naming convention to be clear in$
 	Count           float32
+	RequestMode     string
+	Var1, Var2      string
```

Comment:
Can we use more descriptive variable names than Var1 and Var2?

Reply:
Yes, I will just use MaxRate, MinRate, etc to store these values.